tcases-lib/src/main/java/org/cornutum/tcases/util/XmlWriter.java

```diff
@@ -125,7 +128,7 @@ public void writeAttribute( String name, String value)
     print( " ");
     print( name);
     print( "=\"");
-    print( value);
+    print( NumericEntityEscaper.below(0x20).translate(StringEscapeUtils.escapeXml11(value)));
```

Comment:
Should add a comment to explain this translation. For example, why are both NumericEntityEscaper and StringEscapeUtils required?